examples/src/main/java/org/spine3/server/storage/jdbc/examples/customquery/query/MyAggregateStorageQueryFactory.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2016, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.server.storage.jdbc.examples.customquery.query;
+
+
+import org.spine3.server.aggregate.Aggregate;
+import org.spine3.server.storage.jdbc.query.CreateTableQuery;
+import org.spine3.server.storage.jdbc.util.DataSourceWrapper;
+
+public class MyAggregateStorageQueryFactory<I>
+        extends org.spine3.server.storage.jdbc.aggregate.query.AggregateStorageQueryFactory<I>{
+
+    public MyAggregateStorageQueryFactory(DataSourceWrapper dataSource,
+                                          Class<? extends Aggregate<I, ?, ?>> aggregateClass) {
+        super(dataSource, aggregateClass);
+    }
+
+    @SuppressWarnings("RefusedBequest")
+    @Override
+    public CreateTableQuery newCreateMainTableQuery() {
+        final MyCreateMainTableQuery.Builder builder = MyCreateMainTableQuery.<I>newBuilder()
+                .setDataSource(getDataSource())
+                .setLogger(getLogger())
+                .setTableName("Main_table")
```

Comment:
This table name is used in other queries, yes? So, if we change it, we have to change ALL queries that use it... Let's use the default one here.